definitions/artifacts/aws-iam-role.json

```diff
@@ -38,7 +38,7 @@
         "external_id": {
           "type": "string",
           "title": "External ID",
-          "description": "An external ID is a piece of data that can be passed to the AssumeRole API of the Security Token Service (STS). You can then use the external ID in the condition element in a role’s trust policy, allowing the role to be assumed only when a certain value is present in the external ID."
```

Comment:
This change is weird. Any idea whats going on here @jaketf ?

Reply:
Its escaping the single quote, not sure we need it to do that.